cmd/kube-events-exporter/main.go

```diff
@@ -42,7 +42,7 @@ func main() {
 
 	err := opts.Parse()
 	if err != nil {
-		klog.Fatalf("Error: %s", err)
+		klog.Fatalf("parse options: %v", err)
```

Comment:
I omitted the error indication in the message because I found it redundant with the log level already indicating that a fatal error occurred. What do you think should be the best approach?